### DISCUSSION TITLE
How to set up "Full Name" for email address.

### DISCUSSION THREAD

**Discussion by arreehm** [docker-mailserver/docker-mailserver#3936] [Q&A] [answered]
Greetings,

How to set up full name for email address in container?

Thank you.

**Comment by polarathene** [comment, marked as answer]
Your question lacks context, what problem do you have? DMS supports adding an email address with `setup email add ...` but unless `SPOOF_PROTECTION=1`, this is only relevant for authentication and you can send as any email address you like.

If you are referring to a friendly name presented with the email that has nothing to do with DMS, your mail client handles that when submitting an email for DMS to send.

**Reply by arreehm** [reply]
Okey, now I understand. I meant friendly name, I didn't know it's like that, even thou I already set it up in client few times setting up programatic emailing. I just somehow realized it only now that it's explicitly stated.
